access/grpc/client.go

```diff
@@ -42,20 +43,50 @@ const CanarynetHost = "access.canary.nodes.onflow.org:9000"
 const MainnetHost = "access.mainnet.nodes.onflow.org:9000"
 const PreviewnetHost = "access.previewnet.nodes.onflow.org:9000"
 
+// ClientOption is a configuration option for the client.
+type ClientOption func (*options) 
+
+type options struct {
+	dialOptions []grpc.DialOption
+	jsonOptions []cdcjson.Option
+}
+
+func DefaultClientOptions() *options {
+	return &options{
+		dialOptions: []grpc.DialOption{grpc.WithTransportCredentials(insecure.NewCredentials())},
+		jsonOptions: []cdcjson.Option{cdcjson.WithAllowUnstructuredStaticTypes(true)},
```

Comment:
Maybe add some newlines to make this more readable. Same below

```suggestion
		dialOptions: []grpc.DialOption{
			grpc.WithTransportCredentials(insecure.NewCredentials()),
		},
		jsonOptions: []cdcjson.Option{
			cdcjson.WithAllowUnstructuredStaticTypes(true),
		},
```